easybuild/easyconfigs/s/Sphinx/Sphinx-2.4.3-GCCcore-8.3.0-Python-3.7.4.eb

```diff
@@ -0,0 +1,102 @@
+easyblock = 'PythonBundle'
+
+name = 'Sphinx'
+version = '2.4.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.sphinx-doc.org'
+description = """Sphinx is a tool that makes it easy to create intelligent and beautiful documentation.
+ It was originally created for the new Python documentation, and it has excellent facilities for the 
+ documentation of Python projects, but C/C++ is already supported as well, and it is planned to add 
+ special support for other languages as well."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+builddependencies = [('binutils', '2.32')]
+
+dependencies = [('Python', '3.7.4')]
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+use_pip = True
+
+exts_list = [
+    ('packaging', '20.1', {
+        'checksums': ['e665345f9eef0c621aa0bf2f8d78cf6d21904eef16a93f020240b704a57f1334'],
+    }),
+    ('MarkupSafe', '1.1.1', {
```

Comment:
@furstj This extension is already include in Python 3.7.4 (exact same version).

`packaging` is there already too, but a slightly older version (19.1), is the newer version strictly necessary?

Reply:
@boegel, it seems that the 2.2.0 version of Sphinx is sufficient for me. So I will close this pull request. I'm sorry for wasting your time and thank you for helpful advices that I will certainly use for next PRs.

Reply:
Absolutely no problem, I understand it's sometimes easy to overlook how things change between easyconfig generations...

Thanks a lot for all your efforts!